### PR TITLE
Added Stringtie and Gffcompare

### PR DESCRIPTION
**Pull request by J35P312**

### CODE REVIEW

File: main.nf [hunk 1]
```diff
@@ -80,6 +80,36 @@ process picard_collectrnaseqmetrics{
...
+    script:
+
+    """
+    stringtie ${bam} -G ${params.gtf} > ${sample}.stringtie.gtf
+    """
+
+}
+
+process gffcompare{
+
```
Comment (jemten) [inline main.nf:100]: Don't you want to publish this file?

File: main.nf [hunk 2]
```diff
@@ -80,6 +80,36 @@ process picard_collectrnaseqmetrics{
...
+    input:
+        tuple val(sample) , file(bam),file(bai)  from stringtie_input
+
+    output:
+        tuple val(sample), file("${sample}.stringtie.gtf") into stringtie_output
+
+    script:
+
+    """
+    stringtie ${bam} -G ${params.gtf} > ${sample}.stringtie.gtf
```
Comment (jemten) [inline main.nf:94]: Should we add the strandedness of the library? We could also add option `-p` to allocate more cpus for the assembly

**Review by jemten — APPROVED** [review]
Some small comments. Otherwise 👍

File: main.nf (hunk 1 above)
Comment (J35P312) [reply main.nf:102]: All shall be published!

File: main.nf (hunk 2 above)
Comment (J35P312) [reply main.nf]: the deed is done.

**Pull request merged**